CCL.Importer/Proxies/ProxyWrangler.cs

```diff
@@ -34,7 +34,15 @@ public void MapProxiesOnPrefab(GameObject prefab)
         {
             foreach (var mapper in proxyMappers)
             {
-                mapper.ReplaceProxies(prefab);
+                mapper.CacheAndReplaceProxies(prefab);
+            }
+            foreach (var mapper in proxyMappers)
+            {
+                mapper.ReplaceProxiesUncached(prefab);
```

Comment:
In theory, but doing them as sequential loops means that when this is called the cache is guaranteed - on the off chance something in here needs it

Reply:
That off chance I believe strengthens the reasoning to get rid of the uncached version. There's no harm in calling uncached in cache, but there is in calling cache in uncached.

Reply:
I don't think you understand how this is getting used?

This is for instances that require special handling and do not turn into a replacement monobehaviour.  For example in the caboose replacer there's a component that turns into attaching the career manager, so there is no replacement MonoBehaviour so it can't "add to the cache".  It's a unique step different from the actual map step of the normal pass.

Reply:
I see. Then I think it should have a different name rather than the opposide of cached, as currently that's the idea it gives.

Reply:
Do you have any suggestions for the name?

Reply:
`ReplaceSpecialProxies()`?

Reply:
I can work with that, I'll update that tonight when I have access to visual studio again